### PR TITLE
Add quit to the app menu.

### PR DESCRIPTION
**Pull request by JasonLG1979**

### CODE REVIEW

**Comment by JasonLG1979** [comment]
@TingPing the reasoning behind this is that it's possible for users in DE's besides GNOME to not have an obvious way to quit Pithos if the hide instead of quit mpris plugin pref is selected.

**Comment by JasonLG1979** [comment]
I've been meaning to add this for a while, but it really doesn't affect me since I use GNOME,lol!!!

File: data/ui/PithosWindow.ui [hunk 1]
```diff
@@ -30,6 +30,13 @@
         <attribute name="label" translatable="yes">_About</attribute>
       </item>
     </section>
+    <section>
+      <item>
+        <attribute name="action">app.quit</attribute>
+        <attribute name="label" translatable="yes">_Quit</attribute>
+            <attribute name="accel">&lt;Primary&gt;q</attribute>
```
Comment (TingPing) [inline data/ui/PithosWindow.ui]:
```suggestion
        <attribute name="accel">&lt;Primary&gt;q</attribute>
```

**Comment by TingPing** [comment]
This broke the keybinding for me.

**Comment by TingPing** [comment]
Moving them all to code works, so I guess I'll just do that.

**Pull request merged**